doc/jsk_pcl_ros/nodes/cluster_point_indices_decomposer.md

```diff
@@ -90,6 +90,7 @@ It also publishes tf of centroids of each cluster and oriented bounding box of t
 * `~use_pca` (Boolean, default: `False`):
 
    Run PCA algorithm on each cluster to estimate x and y direction.
+   In case of `~align_boxes` is `False`, The z-axis indicates the first principal component and y-axis indicates the second one.
```

Comment:
Why `The z-axis indicates the first principal component and y-axis indicates the second one`?
In case of `~align_boxes` is `True`, x-axis is the first principal component and y-axis is the second one.

Reply:
OK. I modified code that x-axis is the first principal component and y-axis is the second one.